### PR TITLE
Feat/add headers to messages

### PR DESCRIPTION
**Pull request by NikolaLohinski**
Hi and thanks for the tool, it is very useful.

Here is a way solve https://github.com/subzerocloud/pg-amqp-bridge/issues/16 partially by adding the ability to forward headers. 

I did not manage to make `./tests/main.rs` work locally. I am still new to rust, but the lib.rs tests are fine.

The second commit is just for debugging purposes, made easier with docker compose. Discard it if you are not interested. 

Thanks again !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ You can specify the routing key with the usual syntax used for topic exchanges.
 NOTIFY pgchannel3, '*.orange|Topic message';
 NOTIFY pgchannel3, 'quick.brown.fox|Topic message';
 NOTIFY pgchannel3, 'lazy.#|Topic message';
+NOTIFY pgchannel3, 'key|message|X-First-Header: value1, value2; X-Second-Header: value3'
```
Comment (ruslantalpa) [inline README.md:86]: i think the headers are in the second position, after the key from what it looks like in the code

Comment (ruslantalpa) [reply README.md:86]: maybe this is also the problem with the tests

**Comment by ruslantalpa** [comment]
the code looks good and useful, thank you.
when you get the time see if you can fix the tests (i think i spotted the problem, look at the comments) then i'll take another pass and merge and release.

Thank you

**Comment by NikolaLohinski** [comment]
Hi sorry for the time it took me to get back to you. Tried several times to fix the e2e tests but was never successful. I was never able to make them run locally in any cases I they may probably be just fine.

Regarding your review, please take another look, I think you are mistaken. The headers are in third position if any therefore to get them we should query the index number 2.

Moreover my changes do not affect the e2e tests since passing no headers results in the same behavior as before.

Thanks again

**Comment by ruslantalpa** [comment]
as i said before in the prev comment, the code looks good, maybei'll refactor a bit.
however it will take me a bit (even if the task is 1-2 hours) since i am in the middle of some other stuff and i would not like to switch context :)

**Comment by jcrombez** [comment]
A very usefull feature, thank you for your contribution @NikolaLohinski
I hope you guys find the time to finalize this.

**Comment by ruslantalpa** [comment]
I've implemented this in the latest version 0.0.8
Credit goes to @NikolaLohinski though i could not merge this pr directly since i had to first upgrade all dependencies and add changes to make them work

the only differrence from this PR is that the headers go before the message, i..e `routingkey|headers|message`

**Pull request closed without merging**